web-client/src/pages/work-view.js

```diff
@@ -0,0 +1,298 @@
+                    answer: "Thank you for your feedback!",
+                    answerDate: "12/12/2020",
+                },
+                {
+                    name: 'Reviewer2',
+                    path: picture,
+                    date: "28/11/2020",
+                    stars: 3,
+                    textReview: 'Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.',
+                    answer: "Thank you for your feedback!",
+                    answerDate: "12/12/2020",
+                },
+                {
+                    name: 'Reviewer3',
+                    path: picture,
+                    date: "27/11/2020",
+                    stars: 2,
+                    textReview: 'Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.',
+                    answer: "Thank you for your feedback!",
+                    answerDate: "12/12/2020",
+                }
+            ],
+            reviewer: 
+            {
+                path: picture,
+                date: "28/11/2020",
+                stars: 2,
+                textReview: "Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.",
+            }
+        }
+    ]
+
+    // by default every version should be closed (exept first one)
+    const [open, setOpen] = useState([true, ...(new Array(versions.length - 1).fill(false))]);
+
+    function handleOnClick(item) {
+        if (!open[item])
+            open[item] = true;
+        else
+            open[item] = false;
+
+        setOpen([...open]);
+    }
+
+    const versionList = versions.map((version, i) => {
+        let reviewsList = [];
+        for(let j = 0; j < versions[i].reviews.length; j++)
+        {
+            if(user === "participant")
+            {
+                reviewsList.push(
+                    <CurrentVersionWithReplyToReview
+                        name={version.reviews[j].name}
+                        path={version.reviews[j].path}
+                        stars={version.reviews[j].stars}
+                        review={version.reviews[j].textReview}
+                        date={version.reviews[j].date}
+                        answer={version.reviews[j].answer}
+                        answerDate={version.reviews[j].answerDate}
+                    />
+                );
+            }
+        }
+        if(user === "reviewer")
+        {
+            reviewsList.push(
+                <CurrentVersion
+                    path={version.reviewer.path}
+                    stars={version.reviewer.stars}
+                    review={version.reviewer.textReview}
```

Comment:
Reviewer should see answers from author :p

Reply:
Done 👍